src/components/ChapterWrapper.tsx

```diff
@@ -100,6 +121,41 @@ const ChapterWrapper = (props: ChapterProps) => {
     return inputWrappers;
   };
 
+  async function applicationExists(docRef: any, userID: string) {
```

Comment:
Why cant async function applicationsExists() be the same as submitApplication()? For me it is a little confusing that it is called applicationsExists() because i would expect a boolean to be returned. In this function the logic of submitting an application acutally happens , so why not put everything from applicationsExists in submitApplications? In that case we do not need two functions, when submitApplications() does nothing but call another function.